1-exercises/B-setting-values/exercise2.js

```diff
@@ -27,6 +27,10 @@ let student = {
 
 // write code here
 
+if (student.attendance >= 90 && student.examScore > 60) {
+  student["hasPassed"] = true;
+}
+
```

Comment:
good job, you can add else for false value too :) . 
you can use ternary operator as well